awswrangler/s3/_read_parquet.py

```diff
@@ -607,7 +610,7 @@ def read_parquet(
             boto3_session=boto3_session,
             s3_additional_kwargs=s3_additional_kwargs,
         )
-    return _union(dfs=[_read_parquet(path=p, **args) for p in paths], ignore_index=None)
+    return _union(dfs=[_read_parquet(path=p, **args) for p in paths], ignore_index=ignore_index)
```

Comment:
Should we do the same on the line 334?

Reply:
Yes probably makes sense to change it there too